etc/docker/Dockerfile.sdk-build

```diff
@@ -0,0 +1,47 @@
+# This Dockerfile is meant to be built on top of an existing image from the base-images directory.
+# Suggested usage, from the SDK root:
```

Comment:
I'm not sure if I understand correctly but "Suggested commands to build and run from the SDK root" might be more clear here?